assets/js/steps/check-in-items.js

```diff
@@ -128,7 +131,7 @@ function CheckInItems({ actionHandler }) {
         <>
             <div className='col-md-9'>
                 <Header
-                    header='Aflever'
+                    header='Hand in'
```

Comment:
Shouldn't this be translated, or is it not presented to the user?

Reply:
Please read the slack commet about fixing missing translations in another PR. And there should be NO danish strings in the code base.